engine/perturb.py

```diff
@@ -102,29 +82,37 @@ def perturb(
     copy_all_files,
 ):  # pylint: disable=unused-argument, too-many-positional-arguments
 
-    for m_id in member_ids:
+    for member_id in member_ids:
 
-        if member_type:
-            m_id_str = member_type + "_" + str(m_id)
-        else:
-            m_id_str = str(m_id)
-
-        perturbed_model_input_dir_member_id = perturbed_model_input_dir.format(
-            member_id=m_id_str
-        )
-
-        data = create_perturb_files(
-            model_input_dir,
-            files,
-            perturbed_model_input_dir_member_id,
-            copy_all_files,
+        perturbed_dir = perturbed_model_input_dir.format(
+            member_id=prepend_type_to_member_id(member_type, member_id)
```

Comment:
To be consistent with the other cases, I would suggest
```suggestion
            complete_member_id=prepend_type_to_member_id(member_type, member_id)
```